### DISCUSSION TITLE
Default categories to import?

### DISCUSSION THREAD

**Discussion by Plutoaurus** [firefly-iii/firefly-iii#7851] [Firefly III] [answered]
Are there any default categories that we are able to import?
Thanks!

**Comment by JC5** [comment, marked as answer]
I'm afraid not, you'll have to get inspired to make them all by yourself ;)

**Reply by Plutoaurus** [reply]
Bummer, I'm used to the cloud apps that have it all done. 
Small price to pay for all the extras here though :)
Thanks!
